calculations/do_calculations.py

```diff
@@ -0,0 +1,57 @@
+def temp_given_month(data):
+    """looks for highest temperature/humidity in a given year and month
+
+    Args:
+        data ([type]): [description]
+    """
+    year = input("Enter Year for Avg temperature and humidity as YYYY ")
+    month = input("Enter Month for Avg temperature and humidity as MM ")
+    if int(month) < 10:
+        month = month[1]
+    count = 0
+    avg_highest_temperature = 0
+    avg_lowest_temperature = 0
+    avg_mean_humidity = 0
+    for date_key in data.keys():
+        if date_key.split('-')[0] == year and date_key.split('-')[1] == month:
+            count += 1
+            avg_highest_temperature += data[date_key]['Max TemperatureC']
+            avg_lowest_temperature += data[date_key]['Min TemperatureC']
+            avg_mean_humidity += data[date_key]['Mean Humidity']
+            
+    print(f"Avg Highest: {avg_highest_temperature//count}C\n")
+    print(f"Avg Lowest: {avg_lowest_temperature//count}C\n")
+    print(f"Avg Humidity: {avg_mean_humidity//count}%\n\n")
+    
+    
+    
+     
```

Comment:
Too many extra lines between functions. We are supposed to have two lines between functions.